app/views/inputDropdown.scala.html

```diff
@@ -0,0 +1,27 @@
+@(field: Field,
+dropdownOptions: Seq[(String, String)],
+ args: (Symbol, Any)*
+)(implicit messages: Messages)
+
+@import views.html.helper._
+@import viewsapi.FormFunctions._
+
+@elements = @{new FieldElements(field.id, field, null, args.toMap, messages) }
+
+<div class="govuk-form-group">
+    <label class="govuk-label" for="@elements.id">
+        @elements.args.get('_label)
+        @elements.args.requiredLabelSuffix()
+    </label>
+    <select
+            class="govuk-select"
+            id="@elements.id"
+            name="@elements.id"
+            @elements.args.requiredInput()>
+        <option value="" selected>Please choose...</option>
```

Comment:
Did Pia have wording for this on her wireframes? I'm just wondering if something like "Select a series" might be more direct.

Reply:
This is the wording that is used on the wireframes: https://projects.invisionapp.com/share/S4TVATFN82M#/screens/383373576

Reply:
OK, let's leave it then. It just sounds odd to me. :slightly_smiling_face: 

And I can't find any examples in GOV.UK because [they usually prefer radio buttons](https://design-system.service.gov.uk/components/select/), which might not be appropriate if we have to show any long series lists.